src/Raven.Studio/typescript/common/notifications/models/operation.ts

```diff
@@ -15,11 +15,12 @@ class operation extends abstractNotification {
     taskType = ko.observable<Raven.Server.Documents.Operations.Operations.OperationType>();
     detailedDescription = ko.observable<Raven.Client.Documents.Operations.IOperationDetailedDescription>();
     
-    startTime = ko.observable<moment.Moment>();
     endTime = ko.observable<moment.Moment>();
-    duration: KnockoutComputed<string>;
-    durationInSeconds: KnockoutComputed<number>;
 
+    startTimeForOperation = ko.observable<moment.Moment>();
+    durationForOperation: KnockoutComputed<string>;
```

Comment:
I think we can revert naming conventions here as well. We are in operation class so we know it is duration *of operation* etc.

Reply:
done